sopel/modules/tld.py

```diff
@@ -32,8 +32,12 @@
 
 DATE_FORMAT = '%Y-%m-%d %H:%M:%S %z'
```

Comment:
Bad news, the `%z` is not valid for Python 2.7. I think it was parsed properly since Python 3.2 only.

Reply:
I checked in [the py2 docs](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior) and `%z` is listed? Not sure what you mean "not valid".